terraform/vault-setup.sh

```
@@ -0,0 +1,250 @@
+EOF
+
+# Write policies to Vault
+vault policy write application app-policy.hcl
+vault policy write readonly readonly-policy.hcl
+
+# Enable Kubernetes authentication for EKS
+echo "Enabling Kubernetes authentication..."
+vault auth enable kubernetes
+
+# Get EKS cluster details
+EKS_CLUSTER_NAME=$(terraform output -raw eks_cluster_name)
```

Shellcheck (0.10.0)

[warning] EKS_CLUSTER_NAME appears unused. Verify use (or export if used externally).

(SC2034)